helix-term/src/keymap.rs

```diff
@@ -146,7 +146,16 @@ pub fn default() -> Keymaps {
         key!('r') => commands::replace,
 
         key!('^') => commands::move_line_start,
+        KeyEvent {
+            code: KeyCode::Home,
+            modifiers: KeyModifiers::NONE
+        } => commands::move_line_start,
+
```

Comment:
I wonder why we didn't go `gl`.

Reply:
I think gl/gi are actually great as well, I feel the entire 'goto mode' of kakoune made a lot of sense in general.

Of course at the end of the day the editor will probably have some configuration files that can be loaded at runtime to adjust key mappings.

I just felt home/end keymaps should be in the default/base configuration.

Reply:
I would be open to `gl` `gi` mappings, was just more familiar with the regex-like start/end

Reply:
I think it makes sense to have some sensible defaults, but then postpone adding too many ways of achieving the same thing. As @itzmjauz said, it's better to let user configuration handle that in the future.